src/transformers/models/nystromformer/modeling_nystromformer.py

```diff
@@ -521,6 +520,10 @@ def _init_weights(self, module):
         elif isinstance(module, nn.LayerNorm):
             module.bias.data.zero_()
             module.weight.data.fill_(1.0)
+        elif isinstance(module, nn.Conv2d):
+            module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
+            if module.bias is not None:
+                module.bias.data.zero_()
```

Comment:
Instead of adding these lines here, you can just do `if isinstance(module, (nn.Linear, nn.Conv2d))` in the first if statement of this function.